src/shared/lib/genomeNexusAnnotationSourcesUtils.ts

```diff
@@ -5,3 +5,5 @@ export function shouldShowMutationAssessor() {
         getServerConfig().show_genomenexus_annotation_sources
     );
 }
+
+export const SHOW_MUTATION_ASSESSOR = shouldShowMutationAssessor();
```

Comment:
Here, we may have the same problem of `serverConfig` not being ready yet, right? Is there a way to make sure that we call the function after `serverConfig` is ready?